test/automation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-automation",
-  "version": "1.71.0",
+  "version": "1.91.0",
```

Comment:
Looks like there's still an unintentional change here.

Reply:
Thanks for noticing. There is apparently some automation that keeps changing that, and which crept in again in the last commit fixing the requested things. I have undone it again.